docs/source/theano.rst

```diff
@@ -0,0 +1,157 @@
+# pymc3 and theano
```

Comment:
nitpicking. pymc3 should be stylized as PyMC3. Similar for other packages (Theano, NumPy and  SciPy)